testproto/internal/grpc/server.go

```diff
@@ -0,0 +1,36 @@
+package grpc
+
+import (
+	"context"
+	"log"
+	"net"
+	entities "testproto/internal/gen/entities"
+
+	"google.golang.org/grpc"
+)
+
+type machineServer struct {
+	entities.UnimplementedMachineServiceServer
+}
+
+func (s *machineServer) Create(ctx context.Context, req *entities.CreateMachine) (*entities.MachineResponse, error) {
+	log.Printf("Received: %v", req.GetName())
+	return &entities.MachineResponse{
```

Comment:
entities is not the best name bc it does not clarify meaning. Probably better to use layer name lile DTO.MachineResponse ot something similar. It is pretty minor thing, but it could improve readability